system/jlib/jstats.cpp

```diff
@@ -2413,7 +2419,15 @@ void CRuntimeStatisticCollection::mergeStatistic(StatisticKind kind, unsigned __
 {
     CRuntimeStatistic * target = queryOptStatistic(kind);
     if (target)
+    {
```

Comment:
This change isn't actually needed for the PR - I thought it might be (I was trying to remove cycles from the accumulated stats, but decided not to for the moment).  It should have no ill effect - it only changes things if the stat would have previously have been lost.